src/ServiceControlInstaller.Engine/Configuration/ServiceControl/UpgradeControl.Details.cs

```diff
@@ -6,18 +6,22 @@ public partial class UpgradeControl
     {
         internal static UpgradeInfo[] details =
         {
-            new UpgradeInfo(new Version(2, 0), new Version(1, 41, 3)) //https://github.com/Particular/ServiceControl/issues/1228
+            new (new Version(2, 0), new Version(1, 41, 3))
             {
                 RecommendedUpgradeVersion = new Version(1, 48, 0),
             },
-            new UpgradeInfo(new Version(3, 0), new Version(2, 1, 3)) //https://github.com/Particular/ServiceControl/issues/1228
+            new (new Version(3, 0), new Version(2, 1, 3))
             {
                 RecommendedUpgradeVersion = new Version(2, 1, 3),
             },
-            new UpgradeInfo(new Version(4, 0), new Version(3, 8, 2)) //https://github.com/Particular/ServiceControl/issues/1228
+            new (new Version(4, 0), new Version(3, 8, 2))
             {
                 RecommendedUpgradeVersion = new Version(3, 8, 2),
+            },
+            new (new Version(5, 0), new Version(4, 32, 3))
+            {
+                RecommendedUpgradeVersion = new Version(4, 32, 3),
```

Comment:
Yesterday we agreed that the backporting of these version checks would be added to a new minor,  already bumping these to 4.33.0. That does mean the backport needs to be merged and released first but that is fine.
```suggestion
            new (new Version(5, 0), new Version(4, 33, 0))
            {
                RecommendedUpgradeVersion = new Version(4, 33, 0),
```